src/plugins/xerces/README.md

```diff
@@ -82,7 +82,7 @@ XSD transformations, schemas or DTDs are not supported yet.
 ```sh
 # Backup-and-Restore:/examples/xercesfile
 
-sudo kdb mount xerces.xml /examples/xercesfile xerces
+sudo kdb mount /xerces.xml /examples/xercesfile xerces
```

Comment:
Looks like the leading `/` before `xerces.xml` causes the problem on the Travis build server. I can reproduce the problem locally after I add the slash character.